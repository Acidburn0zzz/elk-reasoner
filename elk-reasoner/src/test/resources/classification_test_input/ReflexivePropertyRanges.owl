Prefix( : = <http://example.org/> )
Prefix( owl: = <http://www.w3.org/2002/07/owl#> )
Prefix( rdfs: = <http://www.w3.org/2000/01/rdf-schema#> )
Ontology(<www.example.org#reflexive_property_ranges>
Annotation(rdfs:label ":S is reflexive since it is a super-property of a reflexive property; since it has range :A then :A should be equivalent to owl:Thing")

SubObjectPropertyOf(:R :S)
ReflexiveObjectProperty(:R)
ObjectPropertyRange(:S :A)

)