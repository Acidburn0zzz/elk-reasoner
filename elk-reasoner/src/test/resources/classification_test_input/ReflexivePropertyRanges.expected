Prefix( : = <http://example.org/> )
Prefix( owl: = <http://www.w3.org/2002/07/owl#> )
Ontology(<www.example.org#reflexive_property_ranges#taxonomy>

EquivalentClasses(:A owl:Thing)

)